### DISCUSSION TITLE
Project `gulp-ava` got archived. Why?

### DISCUSSION THREAD

**Discussion by adymorz** [avajs/ava#2798] [General]
I just noticed that the repo of the `gulp-ava` project got archived, although it is referenced in the readme of the AVA repo. Why?

**Comment by novemberborn** [comment]
The reference is an oversight and should be removed.

I can't recall when I archived this, but even at that point `gulp-ava` would have been unmaintained. If you want you could fork the repository? I'd be happy to transfer the npm package to you as long as it's clear it's not an "official" AVA project.
